src/components/table/table.js

```diff
@@ -1005,6 +1013,21 @@ class Table extends React.Component {
   }
 
   /**
+   * Returns the caption prop wrapped in a <caption> tag,
+   * or null if no caption prop was given.
+   *
+   * @method caption
+   * @return {Object} JSX
+   */
+  get caption() {
+    if (this.props.caption) {
```

Comment:
Our best practice advice would be to not use getters. The file is already full of them I know, so not a blocker.